ateam-common-packets/build.rs

```diff
@@ -0,0 +1,32 @@
+extern crate bindgen;
+
+use std::fs;
+use std::path::PathBuf;
+
+fn main() {
+    // Tell cargo to invalidate the built crate whenever the wrapper changes
+    println!("cargo:rerun-if-changed=include/");
+
+    // The bindgen::Builder is the main entry point
+    // to bindgen, and lets you build up options for
+    // the resulting bindings.
+    let bindings = bindgen::Builder::default()
+        // The input header we would like to generate
+        // bindings for.
+        .header("include/stspin.h")
+        // Tell cargo to invalidate the built crate whenever any of the
+        // included header files changed.
+        .parse_callbacks(Box::new(bindgen::CargoCallbacks))
+        // Finish the builder and generate the bindings.
+        .generate()
+        // Unwrap the Result and panic on failure.
+        .expect("Unable to generate bindings");
+
+    // Write the bindings to the $OUT_DIR/bindings.rs file.
+    let src_dir = "rust-lib";
+    fs::create_dir_all(src_dir).expect("Failed to create src dir.");
+
+    let out_path = PathBuf::from(src_dir);
+    bindings.write_to_file(out_path.join("stspin-bindings.rs"))
```

Comment:
This should probably be `stspin_bindings.rs`, I think rust file names are all supposed snake_case (and I don't think you can create modules with kebab-case)

Reply:
Agreed.